lib/index.js

```diff
@@ -124,13 +115,8 @@ exports.register = (server, options, next) => {
       headers['X-Rate-Limit-Reset'] = rate.reset;
     }
 
-    return reply.continue();
+    return h.continue;
   });
-
-  next();
-
 };
 
-exports.register.attributes = {
-  name: 'hapi-rate-limiter'
-};
+exports.pkg = require('../package.json');
```

Comment:
you can also specify `exports.name = 'hapi-rate-limiter'` if you want parity

Reply:
the `.pkg.name` and `.pkg.version` get hoisted up to `.name` and `.version`

Reply:
oh that's awesome!